tools/createTask.ts

```diff
@@ -53,35 +53,60 @@ export function createIsolateTasksFromConfig(config: IsolateBuildConfig) {
 
   for (const build of config.builds) {
     const isolateCjsBuildTask = gulpIsolateFactory(
-      'cjs',
       {
         indexFile: 'src/index.ts',
         isolateBuildConfig: config,
         buildName: build.name,
         globs: 'src/**/*.ts',
         opts: { base: 'src' }
       },
-      `${build.name}/dist/cjs`,
+      `dist/isolate/${build.name}/dist/cjs`,
       {
-        module: 'ES2015',
-        declaration: true
+        format: 'cjs',
+        overrideSettings: {
+          module: 'ES2015',
+          declaration: true
+        }
       }
     );
 
     const isolateEsmBuildTask = gulpIsolateFactory(
-      'esm',
       {
         indexFile: 'src/index.ts',
         isolateBuildConfig: config,
         buildName: build.name,
         globs: 'src/**/*.ts',
         opts: { base: 'src' }
       },
-      `${build.name}/dist/esm`,
+      `dist/isolate/${build.name}/dist/esm`,
       {
-        target: 'ES5',
-        module: 'ES2015',
-        declaration: true
+        format: 'esm',
+        overrideSettings: {
+          target: 'ES5',
+          module: 'ES2015',
+          declaration: true
+        }
+      }
+    );
+
+    const isolateUmdBuildTask = gulpIsolateFactory(
+      {
+        indexFile: 'src/index.ts',
+        isolateBuildConfig: config,
+        buildName: build.name,
+        globs: 'src/**/*.ts',
+        opts: { base: 'src' }
+      },
+      `dist/isolate/${build.name}/dist/umd`,
+      {
+        format: 'umd',
+        overrideSettings: {
+          target: 'ES5'
+        },
+        sourceMap: false,
+        mangling: false,
+        generateMin: true,
+        outputFileName: 'js-sdsl.js'
```

Comment:
Why this name?

Reply:
because i think

<script src="https://unpkg.com/@js-sdsl/stack/dist/umd/js-sdsl.min.js"></script>
is make sense than
<script src="https://unpkg.com/@js-sdsl/stack/dist/umd/js-sdsl-stack.min.js"></script>

Reply:
Refer to https://www.jsdelivr.com/package/npm/prismjs?path=components.

`https://unpkg.com/@js-sdsl/stack/dist/umd/js-sdsl.min.js` will be ambiguous.

Reply:
I think the name should be discussed.

Reply:
I think `<script src="https://unpkg.com/@js-sdsl/stack/dist/umd/stack.min.js"></script>` wll be better.

Reply:
request has been reflected